src/gc-page-profiler.h

```diff
@@ -3,7 +3,7 @@
 #ifndef GC_PAGE_PROFILER_H
 #define GC_PAGE_PROFILER_H
 
-#include "gc.h"
+#include "gc-stock.h"
```

Comment:
As with the previous comment -- isn't this stock GC specific? Does it need some `#ifndef`?

Reply:
Yes. It's stock GC specific. Resolved.